walkRpde.js

```diff
@@ -85,24 +86,40 @@ async function downloadPage({ apiKey, outputFilePath, indexFilePrefix, indexFile
   const indexFilename = `${outputFilePath}${indexFilePrefix}-index.html`;
   const requestDelaySeconds = Number(requestDelaySecondsStr);
 
+
   // ## Wipe existing output directory
-  await fsPromises.rmdir(outputFilePath, { recursive: true });
-  await fsPromises.mkdir(outputFilePath);
+  if (await fs.pathExists(outputFilePath)) {
```

Comment:
Was having some trouble if either the folder didn't exist or if it had stiff in it